src/app/core/models/common/export-settings.model.ts

```
@@ -21,140 +22,165 @@ export type ExportSettingOptionSearch = {
+      excludedPostedAtOption = null;
     }
 
-    static filterDateOptions(exportDateType: ExportDateType, dateOptions: SelectFormOption[]){
-      const dateOptionToRemove = exportDateType;
-      const filteredOptions = dateOptions.filter(option => option.value !== dateOptionToRemove);
-      return filteredOptions;
+    // Array of unwanted dates
+    const dateOptionsToBeExcluded = [excludedSpendDateOption, excludedPostedAtOption].concat(excludedApprovedOrVerifiedOption);
+
+    // Get base date options
+    const unfilteredDateOptions = this.getExpenseGroupingDateOptions();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)